cloudevents/bindings/http-protocol-binding.md

```diff
@@ -401,14 +401,12 @@ Content-Length: nnnn
 
 In the _batched_ content mode several events are batched into a single HTTP
 request or response body. The chosen [event format](#14-event-formats) MUST
-define how a batch is represented. Based on the [JSON format][json-format] (that
-MUST be supported by any compliant implementation), the [JSON Batch
-format][json-batch-format] is an event format that supports batching.
+define how a batch is represented, including a suitable media type.
```

Comment:
Just to bring this to people attention... we're dropping the MUST here but I think it's safe since we're just repeating what is said [here](https://github.com/cloudevents/spec/blob/9bcaad84435854fc55a852c4ca34533be15bc4ba/cloudevents/bindings/http-protocol-binding.md#14-event-formats).

Reply:
Yup, the MUST in this sentence was a bit odd as it wasn't the main "thrust" of the sentence.